### DISCUSSION TITLE
[Feature Request] Columns Sets: Customize spetials Columns Sets like "Search, Double Files"

### DISCUSSION THREAD

**Discussion by Mnior** [doublecmd/doublecmd#1272] [Suggestions]
DS sets up column sets for special panels such as Search Results. But sometimes it is necessary to expand or change these settings, which is not possible. There are at least two improvements (in order of importance):
1. Expand list of DC() columns, such as “Group of duplicates” (When searching for identical files).
2. Mark column sets for a special operations (Search Results, Duplicates Search Results).

**Comment by zxcat** [comment]
yeah, setting custom column view hides `Group` column and there is no way to get it back. `Default` column view doesn't help. Can this be considered a bug?

Sometimes need this feature too

**Reply by alexx2000** [reply]
No, it is a very special column.

**Comment by advice2020** [comment]
I was going to mention this feature request as well, I agree this would be useful when using the Search Tools "Duplicate Files" feature.
Only other thing I was going to mention was to also maybe add the ability for user to determine the default columns that the "Duplicate Files" uses when applied to "File Box" (to DC file pane).
Currently: Name | Group | Path
User could choose the order of these columns and or add new columns desired to be seen by default.
Not sure if anyone else would find this useful as well along with the other requests from the users above.
But agree it is unfortunate to have to lose the Group number when a different column set is applied, but can understand, not sure how those Group numbers would be stored to get them back.

Thank You
